pages/partials/navTabs/instructorAssessment.ejs

```diff
@@ -28,6 +28,13 @@
     urlSuffix:      '/assessment/'+assessment.id+'/question_statistics',
     iconClasses:    'fas fa-table',
     tabLabel:       'Question stats'}) %>
+<% if (course.manual_grading_visible) { %>
+    <%- include('navTab', {
+        activeSubPage:  'manual_grading',
+        urlSuffix:      '/assessment/'+assessment.id+'/manual_grading',
+        iconClasses:    'fas fa-exclamation',
```

Comment:
The exclamation looks weird. Maybe `fa-pen-alt` or `fa-marker`?

Reply:
Sounds good! Wasn't completely sure which icon to go with since we have so many different "pen on paper" icons yet, so was treating the exclamation as a placeholder. I've updated to `fa-marker`.